pkg/virt-operator/resource/apply/delete.go

```diff
@@ -464,13 +464,64 @@ func DeleteAll(kv *v1.KubeVirt,
 		}
 	}
 
+	if err = deleteKubeVirtLabelsFromNodes(clientset, "kubevirt.io/"); err != nil {
+		return err
+	}
+
 	err = deleteDummyWebhookValidators(kv, clientset, stores, expectations)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+func deleteKubeVirtLabelsFromNodes(clientset kubecli.KubevirtClient, kubeVirtLabelPrefix string) error {
```

Comment:
@fossedihelm i passed `kubeVIrtLabelPrefix` in the function  on the recommdation of @alicefr . If you want me to drop it  , sure i'll do it .

Reply:
I think that @alicefr wants the strings to be plain used in the for loop:
```
if strings.HasPrefix(labelkey, "kubevirt.io/") {
	delete(labels, labelkey)
	modified = true
}
```
Please keep me honest.
Anyway, if the server-side filtering is accepted this is a non-problem since for sure it will not be necessary anymore :)

Reply:
@victortoso ok